cmd/root/bundle_test.go

```diff
@@ -68,157 +63,164 @@ workspace:
-	assert.Equal(t, "PROFILE-1", client.Config.Profile)
-}
-
-func TestBundleConfigureWithMismatchedProfileEnvVariable(t *testing.T) {
-	testutil.CleanupEnvironment(t)
-
-	t.Setenv("DATABRICKS_CONFIG_PROFILE", "PROFILE-1")
-	cmd := emptyCommand(t)
-	b := setupWithHost(t, cmd, "https://x.com")
-
-	_, err := b.InitializeWorkspaceClient()
-	assert.ErrorContains(t, err, "config host mismatch: profile uses host https://a.com, but CLI configured to use https://x.com")
-}
-
-func TestBundleConfigureWithProfileFlagAndEnvVariable(t *testing.T) {
-	testutil.CleanupEnvironment(t)
-
-	t.Setenv("DATABRICKS_CONFIG_PROFILE", "NOEXIST")
-	cmd := emptyCommand(t)
-	err := cmd.Flag("profile").Value.Set("PROFILE-1")
-	require.NoError(t, err)
-	b := setupWithHost(t, cmd, "https://a.com")
-
-	client, err := b.InitializeWorkspaceClient()
-	require.NoError(t, err)
-	assert.Equal(t, "https://a.com", client.Config.Host)
-	assert.Equal(t, "PROFILE-1", client.Config.Profile)
-}
-
-func TestBundleConfigureProfileDefault(t *testing.T) {
-	testutil.CleanupEnvironment(t)
-
-	// The profile in the databricks.yml file is used
-	cmd := emptyCommand(t)
-	b := setupWithProfile(t, cmd, "PROFILE-1")
-
-	client, err := b.InitializeWorkspaceClient()
-	require.NoError(t, err)
-	assert.Equal(t, "https://a.com", client.Config.Host)
-	assert.Equal(t, "a", client.Config.Token)
-	assert.Equal(t, "PROFILE-1", client.Config.Profile)
-}
-
-func TestBundleConfigureProfileFlag(t *testing.T) {
-	testutil.CleanupEnvironment(t)
-
-	// The --profile flag takes precedence over the profile in the databricks.yml file
-	cmd := emptyCommand(t)
-	err := cmd.Flag("profile").Value.Set("PROFILE-2")
-	require.NoError(t, err)
-	b := setupWithProfile(t, cmd, "PROFILE-1")
-
-	client, err := b.InitializeWorkspaceClient()
-	require.NoError(t, err)
-	assert.Equal(t, "https://a.com", client.Config.Host)
-	assert.Equal(t, "b", client.Config.Token)
-	assert.Equal(t, "PROFILE-2", client.Config.Profile)
-}
-
-func TestBundleConfigureProfileEnvVariable(t *testing.T) {
-	testutil.CleanupEnvironment(t)
-
-	// The DATABRICKS_CONFIG_PROFILE environment variable takes precedence over the profile in the databricks.yml file
-	t.Setenv("DATABRICKS_CONFIG_PROFILE", "PROFILE-2")
-	cmd := emptyCommand(t)
-	b := setupWithProfile(t, cmd, "PROFILE-1")
-
-	client, err := b.InitializeWorkspaceClient()
-	require.NoError(t, err)
-	assert.Equal(t, "https://a.com", client.Config.Host)
-	assert.Equal(t, "b", client.Config.Token)
-	assert.Equal(t, "PROFILE-2", client.Config.Profile)
-}
-
-func TestBundleConfigureProfileFlagAndEnvVariable(t *testing.T) {
-	testutil.CleanupEnvironment(t)
-
-	// The --profile flag takes precedence over the DATABRICKS_CONFIG_PROFILE environment variable
-	t.Setenv("DATABRICKS_CONFIG_PROFILE", "NOEXIST")
-	cmd := emptyCommand(t)
-	err := cmd.Flag("profile").Value.Set("PROFILE-2")
-	require.NoError(t, err)
-	b := setupWithProfile(t, cmd, "PROFILE-1")
+func TestBundleConfigureProfile(t *testing.T) {
```

Comment:
While acceptance tests would be better suited for most of these, since this PR changes how auth is loaded I opted to keep the unit tests.